moveit_core/CMakeLists.txt

```diff
@@ -226,6 +231,32 @@ add_subdirectory(collision_distance_field)
 add_subdirectory(kinematics_metrics)
 add_subdirectory(dynamics_solver)
 
+set(pymoveit_libs
+  moveit_collision_detection
+  moveit_kinematic_constraints
+  moveit_planning_scene
+  moveit_python
+  moveit_robot_model
+  moveit_robot_state
+  moveit_transforms
+)
+
+if(pybind11_catkin_FOUND)
```

Comment:
pybind11 was declared as required dependency above. No need for this if. But doesn't harm as well.

Reply:
I thought it better to make it optional at least to start, so I'm going to remove the "required" rather than the if. But I think either way is totally reasonable